packages/text/README.md

```diff
@@ -0,0 +1,37 @@
+![logo-farmblocks](https://user-images.githubusercontent.com/7760/31051341-4d280118-a63c-11e7-9e8f-3b375ca8f9a0.png)
+
+# Farmblocks-Theme
+
+React components for displaying text. [Storybook](https://cravefood.github.io/farmblocks/)
+
+## Design spec
+See it on zeplin:
+[Title](https://scene.zeplin.io/project/595a9cd3b401bf1876faab27/screen/59f0dee4b4c8ae0eea8769e4)
+[Caption](https://scene.zeplin.io/project/595a9cd3b401bf1876faab27/screen/59f0dee25617ae2c1f2fcb26)
+[Paragraph](https://scene.zeplin.io/project/595a9cd3b401bf1876faab27/screen/59f0dee4c7570f862449ae63)
+
+## Usage
+
+```jsx
+import React from 'react';
+import { render } from 'react-dom';
+import { Title, Caption, Paragraph, fontSizes, fontTypes } from '@crave/farmblocks-text';
+
+render(
+  <div>
+    <Title size={fontSizes.X_LARGE} type={fontTypes.NORMAL}>Content title </Title>
```

Comment:
I was thinking about EXTRA_LARGE vs X_LARGE...

On the Image component, we went with EXTRA, but X gives us the advantage of being able to use XX_SMALL and XXX_SMALL in the future, similar to the css api...

The other argument would be that size names are more semantic and more than 5 variations might be too much, so TINY and HUGE as aliases for XX on each direction can be enough, and if the need for XXX appears we could use EXTRA_TINY and EXTRA_HUGE

I personally dont have a strong preference on either choice, but it would be nice to use the same choice for different components, for consistence sake

Reply:
We went with the X_LARGE option.